staging/src/k8s.io/api/resource/v1/types.go

```diff
@@ -1318,7 +1318,7 @@ type DeviceToleration struct {
 	// Effect indicates the taint effect to match. Empty means match all taint effects.
 	// When specified, allowed values are NoSchedule and NoExecute.
 	//
-	// +optional
+	// +required
```

Comment:
This is the `DeviceToleration`, where "Empty means match all taint effects.". Here `+optional` is correct.

I thought it was `+optional` by mistake in `DeviceTaint`, but looking again now I see that it's correctly marked `+required` there, in all versions.

So nothing to fix?